src/ray/core_worker/core_worker.cc

```diff
@@ -238,6 +249,24 @@ void CoreWorkerProcess::RunTaskExecutionLoop() {
   instance_.reset();
 }
 
+void CoreWorkerProcess::RunStatsService() {
+  absl::MutexLock lock(&CoreWorkerProcess::stats_mutex_);
+  // Initialize stats.
+  const ray::stats::TagsType global_tags = {{ray::stats::ComponentKey, "core_worker"},
+                                            {ray::stats::VersionKey, "0.9.0.dev0"}};
+
+  // NOTE(lingxuan.zlx): We assume RayConfig is initialized before it's used.
+  // RayConfig is generated in Java_io_ray_runtime_RayNativeRuntime_nativeInitialize
+  // for java worker or in constructor of CoreWorker for python worker.
+  ray::stats::Init(global_tags, RayConfig::instance().metrics_agent_port(),
```

Comment:
The design may change in the future. I'm considering removing the initialization of RayConfig in JNI. Could you please add `RAY_CHECK(RayConfig::instance().metrics_agent_port() != 0)` before this line?

Reply:
> The design may change in the future. I'm considering removing the initialization of RayConfig in JNI. Could you please add `RAY_CHECK(RayConfig::instance().metrics_agent_port() != 0)` before this line?

So will we get Rayconfig initialized in core worker constructor as well, just like python worker?

Reply:
Sure you can.